dev-python/pyzbar/pyzbar-0.1.5.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+inherit distutils-r1
+
+DESCRIPTION="Read one-dimensional barcodes and QR codes from Python."
+HOMEPAGE="https://github.com/NaturalHistoryMuseum/pyzbar/"
+SRC_URI="https://github.com/NaturalHistoryMuseum/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="
+	media-gfx/zbar
+	dev-python/pillow[${PYTHON_USEDEP}]
+	virtual/python-enum34[${PYTHON_USEDEP}]
+	virtual/python-pathlib[${PYTHON_USEDEP}]
+"
+
+DEPEND="
+	${RDEPEND}
+	dev-python/setuptools[${PYTHON_USEDEP}]
+	test? (
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/mock[${PYTHON_USEDEP}]
+	)
+"
+
+python_test() {
+	nosetests -v || die
```

Comment:
You don't depend on nose. Also, it doesn't seem that the tests actually require nose, so please run them via built-in unittest if possible.

https://wiki.gentoo.org/wiki/Project:Python/Tests